src/main/java/com/zalando/ecommerce/dto/CartRequest.java

```diff
@@ -0,0 +1,13 @@
+package com.zalando.ecommerce.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter @Setter
+@NoArgsConstructor @AllArgsConstructor
```

Comment:
format them in a new line .. as
```suggestion
@NoArgsConstructor 
@AllArgsConstructor
```

Reply:
done 👍